spec/features/bucket_lists/user_deletes_bucket_list_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+feature 'authenticated user deletes a bucket list', %(
+  As a last minute traveler or traveler who's stay is unexpectedly extended,
+  I want to delete a bucket list
+  So I can throw away my dreams.
+) do
+# Acceptance Criteria
+#[] I can click a link to delete a bucket list
+#[] I receive an success message after deleting a bucket list
```

Comment:
Incorrect indentation detected (column 0 instead of 2).<br>Missing space after #.